apps/main/src/dex/components/PagePool/PoolDetails/PoolStats/Rewards.tsx

```diff
@@ -31,8 +31,10 @@ const Rewards = ({ chainId, poolData, rewardsApy }: RewardsProps) => {
   const { base, other } = rewardsApy ?? {}
   const { haveBase, haveOther, haveCrv } = haveRewardsApy(rewardsApy ?? {})
   const network = useStore((state) => state.networks.networks[chainId])
-  const { data: campaigns } = useCampaignsByNetwork(network.networkId as Chain)
-  const campaignRewardsPool = campaigns?.[poolData.pool.address]
+  const { data: campaigns } = useCampaignsByAddress({
+    blockchainId: network.networkId as Chain,
+    address: poolData.pool.address as Address,
```

Comment:
`as Address` every is annoying as hell to me, but keeping it as `string` is bad practice too imo